src/LibVLCSharp.MAUI/LibVLCSharp.MAUI.csproj

```diff
@@ -16,8 +16,49 @@ LibVLC needs to be installed separately, see VideoLAN.LibVLC.* packages.
     </Description>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net8.0-android'">
+    <DefineConstants>ANDROID</DefineConstants>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net8.0-ios'">
+    <DefineConstants>IOS</DefineConstants>
+  </PropertyGroup>
```

Comment:
you do not need those, already defined in `Directory.Build.props`

Reply:
It's probably the Upgrade assistant that added them. Removed.